app/helpers/application_helper.rb

```diff
@@ -113,6 +113,39 @@ def fa_icon(icon, attributes = {})
     content_tag(:i, nil, attributes.merge(class: class_names.join(' ')))
   end
 
+  def material_symbol(icon, attributes = {})
+    return awesome_icon(material_to_awesome(icon), attributes) if current_flavour == 'polyam'
```

Comment:
One idea I have for this is instead of explicitly checking for the polyam flavour to allow selecting icons with the theming system.
Maybe an attribute like `icon-theme` in theme.yml files